projects/storefrontstyles/scss/_page-template.scss

```diff
@@ -15,15 +15,15 @@ $page-template-whitelist: LandingPage2Template, ContentPage1Template,
   StoreFinderPageTemplate, CheckoutLoginPageTemplate, AccountPageTemplate !default;
 
 @each $selector in $page-template-whitelist {
-  @include forVersion(0, 2) {
+  @include forVersion(0, 3) {
```

Comment:
We should drop the forVersion completely for the next 3.0 release. 

The 0 - 2 variation should no longer be used, as it conflicts with our idea to make the page layout related  to the page-template.

the 2.1 variation should become default now